build.jam

```diff
@@ -5,24 +5,27 @@
 
 require-b2 5.2 ;
 
+constant boost_dependencies :
```

Comment:
Too many dependencies. It should be

    assert
    config
    core
    move
    static_assert
    throw_exception
    type_traits